spec/models/form_document/content_spec.rb

```diff
@@ -28,7 +28,7 @@
   end
 
   it "has all form attributes the original form has" do
-    expected_attributes = form.attributes.except(*%w[id state external_id pages question_section_completed declaration_section_completed share_preview_completed])
+    expected_attributes = form.attributes.except(*%w[id state external_id pages question_section_completed declaration_section_completed share_preview_completed welsh_completed])
```

Comment:
I think `welsh_completed` should be added to Form#ATTRIBUTES_NOT_IN_FORM_DOCUMENT too?

Reply:
Thanks for spotting!